content/en/posts/eth2-staking-with-k8s-prysm.md

```diff
@@ -468,19 +472,19 @@ On your master:
 2. Check the log of the beacon node.
 
     ```bash
-    microk8s kubectl logs -f -nprysm -lapp=beacon
+    microk8s kubectl logs -f -nprysm -l app=beacon
```

Comment:
The commands I use all include `-lapp` so I think it works without the space. If the purpose is to clearly specify which flag the commands use, should we also change other flags as well? E.g. `-nprysm` to `-n prysm`.

Reply:
I just saw `-l app=<validator name>` used in the following. If the purpose is to let readers quickly find the app name for different validators, then I'm fine with this change (i.e. adding space after -l flag)